python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -173,6 +173,10 @@ cdef class Column:
             children,
         )
 
+    cpdef Column column_from_self_view(self):
+        """Return a new column from self.view()."""
+        return Column.from_libcudf(move(make_unique[column](self.view())))
```

Comment:
*question*: shall we call this what it is? `cpdef Column copy(self)` ? Or possibly `deepcopy`?

Reply:
Oh wow, this made me realize pylibcudf Column already has a `copy` method. Thanks I was able to not reinvent the wheel here then